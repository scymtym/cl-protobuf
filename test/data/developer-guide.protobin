
“
devloper-guide.protodeveloper_guide"
Test1	
a ("
Test2	
b (	"*
Test3!
c (2.developer_guide.Test1"
Test4
d (B