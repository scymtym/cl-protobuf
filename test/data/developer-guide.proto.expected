// developer-guide.proto
package developer_guide;
message Test1 {
  required int32 a = 1;
  }
message Test2 {
  required string b = 2;
  }
message Test3 {
  required .developer_guide.Test1 c = 3;
  }
message Test4 {
  repeated int32 d = 4 [packed = true];
  }
